scroll-to-text-fragment/stash.py

```diff
@@ -1,13 +1,13 @@
 import time
 
 def main(request, response):
-    key = request.GET.first("key")
+    key = request.GET.first(b"key")
 
-    if request.method == "POST":
+    if request.method == u"POST":
```

Comment:
This is a bit weird as I noted in the other PR, but I guess it's an upstream problem?

Reply:
Added. Thank you!

Reply:
I think you meant to leave the "Added. Thank you!" reply on the other comment Ziran?

For this, cc @Hexcles as an FYI

Reply:
@stephenmcgruer: yes, I meant to leave the reply to the other comment. Thanks for spotting this :).

Reply:
https://github.com/web-platform-tests/wpt/issues/23362 is the tracking issue.